webapp/src/ts/components/sidebar-menu/sidebar-menu.component.ts

```diff
@@ -0,0 +1,167 @@
+  }
+
+  close() {
+    return this.globalActions.closeSidebarMenu();
+  }
+
+  replicate() {
+    if (this.replicationStatus?.current?.disableSyncButton) {
+      return;
+    }
+    return this.dbSyncService.sync(true);
+  }
+
+  logout() {
+    this.modalService.show(LogoutConfirmComponent);
+  }
+
+  private subscribeToRouter() {
+    const routerSubscription = this.router.events
+      .pipe(filter(event => event instanceof NavigationStart))
+      .subscribe(() => this.close());
+    this.subscriptions.add(routerSubscription);
+  }
+
+  private subscribeToStore() {
+    const subscribeReplicationStatus = this.store
+      .select(Selectors.getReplicationStatus)
+      .subscribe(replicationStatus => this.replicationStatus = replicationStatus);
+    this.subscriptions.add(subscribeReplicationStatus);
+
+    const subscribeSidebarMenu = this.store
+      .select(Selectors.getSidebarMenu)
+      .subscribe(sidebarMenu => this.sidebar?.toggle(sidebarMenu?.isOpen));
+    this.subscriptions.add(subscribeSidebarMenu);
+
+    const subscribePrivacyPolicy = this.store
+      .select(Selectors.getShowPrivacyPolicy)
+      .subscribe(showPrivacyPolicy => this.setSecondaryOptions(showPrivacyPolicy));
+    this.subscriptions.add(subscribePrivacyPolicy);
+  }
+
+  private openFeedback() {
+    this.modalService.show(FeedbackComponent);
+  }
+
+  private setModuleOptions() {
+    this.moduleOptions = [
+      {
+        routerLink: 'messages',
+        icon: 'fa-envelope',
+        translationKey: 'Messages',
+        hasPermissions: 'can_view_messages,!can_view_messages_tab'
+      },
+      {
+        routerLink: 'tasks',
+        icon: 'fa-flag',
+        translationKey: 'Tasks',
+        hasPermissions: 'can_view_tasks,!can_view_tasks_tab'
+      },
+      {
+        routerLink: 'reports',
+        icon: 'fa-list-alt',
+        translationKey: 'Reports',
+        hasPermissions: 'can_view_reports,!can_view_reports_tab'
+      },
+      {
+        routerLink: 'contacts',
+        icon: 'fa-user',
+        translationKey: 'Contacts',
+        hasPermissions: 'can_view_contacts,!can_view_contacts_tab'
+      },
+      {
+        routerLink: 'analytics',
+        icon: 'fa-bar-chart-o',
+        translationKey: 'Analytics',
+        hasPermissions: 'can_view_analytics,!can_view_analytics_tab',
+      },
+    ];
+  }
+
+  private setSecondaryOptions(showPrivacyPolicy = false) {
+    this.secondaryOptions = [
+      {
+        routerLink: 'about',
+        icon: 'fa-question',
+        translationKey: 'about',
+        canDisplay: true,
+      },
+      {
+        routerLink: 'user',
+        icon: 'fa-user',
+        translationKey: 'edit.user.settings',
+        hasPermissions: 'can_edit_profile'
+      },
+      {
+        routerLink: 'privacy-policy',
+        icon: 'fa-lock',
+        translationKey: 'privacy.policy',
+        canDisplay: showPrivacyPolicy,
+      },
+      {
+        icon: 'fa-bug',
+        translationKey: 'Report Bug',
```

Comment:
Ah good eye! In Figma, the icons don't seem consistently left, right, or centered aligned. Maybe because they all have different shapes (They all already have the same size)

I'm going to center aligned them, since they have different shapes

Reply:
Fixed! 
<img width="300" alt="Screenshot 2024-09-03 at 5 06 39 PM" src="https://github.com/user-attachments/assets/90a3ecdd-39b8-4d96-a43c-3ea7e2ebdfe7">